src/main/java/com/beust/jcommander/UnixStyleUsageFormatter.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (C) 2010 the original author or authors.
+ * See the notice.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.beust.jcommander;
+
+import java.util.EnumSet;
+import java.util.List;
+
+/**
+ * A unix-style usage formatter.
+ */
+public class UnixStyleUsageFormatter extends DefaultUsageFormatter {
```

Comment:
Let's start with an interface `IUsageFormatter` that captures the methods that a formatter needs to implement, then let's have `DefaultUsageFormatter` and `UnixStyleUsageFormatter` implement it.

Reply:
I have addressed this.

Reply:
Thanks, looking good. I don't see any specific test for this new feature though, think you could write a few?

Reply:
I'll look into it.